static-tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+echo "Running static test - gofmt"
+GOFMT_FILES=$(gofmt -l .)
+if [ -n "${GOFMT_FILES}" ]; then
+  printf >&2 'gofmt failed for the following files:\n%s\n\nplease run "gofmt -w ." on your changes before committing.\n' "${GOFMT_FILES}"
+  TEST_STATUS=FAIL
+fi
+
+echo "Running static test - golint"
+GOLINT_ERRORS=$(golint ./... | grep -v "Id should be")
```

Comment:
Just checking: do we still want to special case this linting message?

Reply:
Removed this.